inbox/events/google.py

```diff
@@ -206,25 +183,25 @@ def _get_resource_list(self, url: str, **params) -> List[Dict[str, Any]]:
                     token = self._get_access_token(force_refresh=True)
                     continue
                 elif r.status_code in (500, 503):
-                    log.warning("Backend error in calendar API; retrying")
+                    self.log.warning("Backend error in calendar API; retrying")
```

Comment:
Notice that this was logging using global logger before, so the account_id info was missing on the logger